pages/components/Alert/alert.css

```diff
@@ -1,16 +1,13 @@
 .alert {
-    background-color: var(--primary-colour);
+    background-color: #cff4fc;
+    font-family: var(--font);
     padding: 1em;
     margin: 1em 0;
-    width: 45em;
     border-radius: 0.5em;
+    width: max-content;
```

Comment:
Hi Rohit, thank you for letting me review your PR, here, alerts can use 100% width because
- Users expect a close button at the end of alert and more often it lies to the right hand side